### ISSUE TITLE
create a synlink

### ISSUE THREAD

**Issue by jonbr** [panitw/easy-rpm#86]
It would be very valuable to be able to use something like 'postInstallScript' to create a symlink

**Comment by peterjanes** [comment]
Agreed on a desire for symlinks. However, [per one of the RPM developers](https://www.spinics.net/lists/rpm/msg07921.html) "As a rule of thumb, when possible its always better to package than do things in install scriptlets." (That is, create the symlinks in the `%build` section of the RPM spec file.) I think symlinks would be better handled as a new property in `files`, or (less ideally) a new `symlinks` option.
